src/app/components/tasks/Tasks/Tasks.tsx

```diff
@@ -69,38 +95,60 @@ const Tasks: React.FC = () => {
   }
 
   const emptyPlaceholder = hasPermission === false && theme === UNDERMINING ? EMPTY_TEXT_NO_PERMISSION : EMPTY_TEXT
+  const enforcementTasksAvailable = !!enforcementDataSource?.results?.length
 
   return (
-    <Row>
-      <Column spanLarge={ 72 }>
-        <TableTasks
-          data={ results || [] }
-          isBusy={ isBusy }
-          onChange={onChangeTable}
-          pagination={{
-            page: pagination.page,
-            pageSize: pagination.pageSize,
-            collectionSize: count || 1
-          }}
-          sorting={ sorting }
-          emptyPlaceholder={ emptyPlaceholder }
-        />
-      </Column>
-      <Column spanLarge={ 28 }>
-        <TasksFilter
-          role={ role }
-          roles={ roles }
-          setRole={ (value: string) => onChangeFilter("role", value) }
-          theme={ theme }
-          themes={ caseThemes?.results }
-          setTheme={ (value: string) => onChangeFilter("theme", value) }
-          setPageSize={ onChangePageSize }
-          pageSize={ pagination.pageSize?.toString() || "10" }
-          owner={ owner }
-          setOwner={ (value: string) => onChangeFilter("owner", value) }
-        />
-      </Column>
-    </Row>
+    <>
+      <Row>
+        <Column spanLarge={ 72 }>
+          { enforcementTasksAvailable ?  (
+            <Wrap>
+              <StyledHeading as="h2"><span>Handhavingsverzoeken </span><EnforcementIcon show /></StyledHeading>
+              <TableTasks
+                data={ enforcementDataSource?.results || [] }
+                isBusy={ isBusy }
+                onChange={onChangeTable}
+                pagination={false}
+                sorting={ sorting }
+                emptyPlaceholder={ emptyPlaceholder }
+                />
+            </Wrap>
+            ) : null
```

Comment:
You can do ` { enforcementTasksAvailable &&  <Wrap> }` instead of ` { enforcementTasksAvailable ?  <Wrap> : null }`